<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>회원탈퇴</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>아이디</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="member_dlt_id">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="member_dlt_pw">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_member_dlt_push">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>회원탈퇴</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_delete_back">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
